C/plugins/north/OMF/omf.cpp

```diff
@@ -5313,13 +5313,11 @@ bool OMF::sendFledgeAssetType()
 	catch (const Unauthorized &e)
 	{
 		Logger::getLogger()->error(MESSAGE_UNAUTHORIZED);
-		
 	}
 	catch (const Conflict& e)
 	{
-		handleRESTException(e, "The OMF endpoint reported a Conflict when sending FledgeAsset Type");
-		Logger::getLogger()->warn(MESSAGE_PI_UNSTABLE, 409);
-		m_PIstable = false;
+		Logger::getLogger()->warn("FledgeAsset Type exists with a different definition");
```

Comment:
Can we give the user any help here? Seeing this in the log it is unclear what, if anything the user can do about it.

Reply:
I am working on this. It's more complicated than I thought. It turns out that if you edit Static Data values, Linked Type instances will update the Static Data values in all AF Element instances. In Complex Types, this doesn't happen.